src/runner.go

```diff
@@ -35,20 +35,20 @@ var verificationArgs = []string{
 
 // Verify that no one tampered with yaml file
 func verifyYamlFile(yamlData []byte) bool {
-
 	if !*config.VerifyYAML {
 		log.Warnln("WARNING: Playbook verification disabled.")
 		return true
 	}
 
-	// --payload here will be a no-op because no upload is performed when using the verifier
-	//   but, it will allow us to update the egg!
-
 	env := os.Environ()
-
 	if !*config.InsightsCoreGPGCheck {
+		log.Infoln("Calling insights-client with --no-gpg to skip signature validation...")
+		// --payload here will be a no-op because no upload is performed when
+		// using the verifier but, it will allow us to update the egg!
 		verificationArgs = append(verificationArgs, "--no-gpg")
 		env = append(env, "BYPASS_GPG=True")
+	} else {
+		log.Infoln("Verifying gpg signature with insights-clieht")
```

Comment:
Hmm the sentence.. I don't know .. would somehow improve it to make it similar in both cases
```suggestion
		log.Infoln("Calling insights-client with gpg signature validation...")
```

I wouldn't be against printing the `verificationArgs` of the command in logs.. if that would make it clear

Reply:
Sounds about right. I will add that.